config/config.go

```diff
@@ -349,20 +355,30 @@ func (c *Config) GetStatsFlushInterval() time.Duration {
 
 // GetCommittedTierFSParams returns parameters for building a tierFS.  Caller must separately
 // build and populate Adapter.
-func (c *Config) GetCommittedTierFSParams() (*pyramidparams.Params, error) {
+func (c *Config) GetCommittedTierFSParams() (*pyramidparams.ExtParams, error) {
 	adapter, err := factory.BuildBlockAdapter(c)
 	if err != nil {
 		return nil, fmt.Errorf("build block adapter: %w", err)
 	}
+	rangePro := viper.GetFloat64(CommittedLocalCacheRangeProportion)
+	metaRangePro := viper.GetFloat64(CommittedLocalCacheMetaRangeProportion)
+
+	if rangePro+metaRangePro != 1 {
```

Comment:
This also makes life difficult, because 10 is divisble by 5 so nice ASCII floats don't sum exactly (e.g. https://play.golang.org/p/f3T-Ccruu7J is a bit scary).  Consider adding some slop, e.g.
```suggestion
	if something.Abs(rangePro+metaRangePro-1.0) > 1e-6 {
```
(I am not sure whether it is possible to create a nice example when summing just 2 numbers, though)